apps/web/test/utils/bookingScenario/bookingScenario.ts

```diff
@@ -663,80 +810,24 @@ export function mockErrorOnVideoMeetingCreation({
   });
 }
 
-export function expectWebhookToHaveBeenCalledWith(
```

Comment:
Moved all expects to separate module so that we can add more expects there easily.